IBPSA/Fluid/Actuators/Dampers/PressureIndependent.mo

```diff
@@ -125,8 +155,22 @@ equation
                                      m_flow_der2=0),
                  y2dd=y2dd)));
   end if;
-
-
+  // Computation of damper opening
+  k = IBPSA.Fluid.BaseClasses.FlowModels.basicFlowFunction_inv(
+    m_flow=m_flow,
+    dp=dp,
+    m_flow_small=1E-2*abs(m_flow_nominal),
+    dp_small=1E-4*dp_nominal_pos);
+  kDam = if dpFixed_nominal > Modelica.Constants.eps then
+    sqrt(1 / (1 / k^2 - 1 / kFixed^2)) else k;
+  // Use of regStep might no longer be needed when the leakage flow modeling is updated.
+  y_actual_smooth = IBPSA.Utilities.Math.Functions.regStep(
+    x=y_internal - y_min,
+    y1=IBPSA.Fluid.Actuators.BaseClasses.exponentialDamper_inv(
+      kThetaSqRt=sqrt(2*rho)*A/kDam, kSupSpl=kSupSpl, ySupSpl=ySupSpl, invSplDer=invSplDer),
```

Comment:
I'm confused by this implementation, specifically the first argument: that input requires something with units of 'k' (judging from the name), but you assign something with units rho*A/k, which seems wrong?

Reply:
`exponentialDamper_inv` uses the square root of the loss coefficient (similar to `exponentialDamper`), `kThetaSqRt = sqrt(dp / (1/2 * rho * v^2))`. The first argument is the inline computation of this value from the flow coefficient `k = m_flow / sqrt(dp)`.

Reply:
@mwetter You are right that for this model it seems better suited to use the actual value of the loss coefficient in `exponentialDamper_inv` and the square value of the flow coefficient in `basicFlowFunction_inv`. However 

- an equation for the actual value of the flow coefficient would still need to be included as the variable is declared in the base class (although it might not require the same smoothness),
- this would introduce some ambiguity on the quantity of the I/O variables between these new  functions and the original functions that they inverse.